pkg/networkmanager/firewall.jsx

```diff
@@ -418,6 +435,19 @@ class AddServicesModal extends React.Component {
                 </Modal.Header>
                 <div id="cockpit_modal_dialog">
                     <Modal.Body id="add-services-dialog">
+                        { firewall.activeZones.size > 1 &&
+                            <React.Fragment>
+                                <form className="ct-form-layout horizontal">
+                                    <label htmlFor="zone-input">{ _("Add services to following zones:") }</label>
+                                    <fieldset id="zone-input">
+                                        { Array.from(firewall.activeZones).map(z =>
```

Comment:
This array should be sorted, both for better presentation and for more predictability in the tests.